sklearn/linear_model/base.py

```diff
@@ -360,6 +426,14 @@ class LinearRegression(LinearModel, RegressorMixin):
 
     normalize : boolean, optional, default False
         If True, the regressors X will be normalized before regression.
+        This parameter is ignored when `fit_intercept` is set to `False`.
```

Comment:
indeed. "same" is not correct.

+1 for

Note that this makes the hyperparameters learnt more robust and almost independent of the number of samples.

thx @MechCoder